src/main/java/com/amplitude/HttpTransport.java

```diff
@@ -89,6 +89,10 @@ public void setCallbacks(AmplitudeCallbacks callbacks) {
           Response response = null;
           try {
             response = httpCall.makeRequest(events);
+            if (logger.getLogMode() == AmplitudeLog.LogMode.DEBUG) {
+                String debugMessage = "Sending " + events.size() + " events. \n" + response.toString();
+                logger.log(logger.getTimeStamp(), debugMessage);
```

Comment:
The tag usage maybe from Android, show class or more meaningful information. I guess we don't need to use timestamp as a tag. If you prefer to have timestamp in debug message, how about just change format inside the logger debug method?